expan/core/statistical_test.py

```diff
@@ -65,7 +69,8 @@ def size(self):
 
 
 class FeatureFilter(JsonSerializable):
-    """ This class represents a filter, restricting a DataFrame to rows with column_value in column_name. 
+    """ This class represents a filter, restricting a DataFrame to rows with column_value in column_name.
+    
```

Comment:
this class can have a method that would filter a given dataframe and then we would apply this method of all filters on our initial dataframes instead of doing it by hand with `data_for_analysis` above

Reply:
fixed!

Reply:
thanks Darya!

Reply:
👍